katas/langtonant/maria/README.md

```diff
@@ -0,0 +1,4 @@
+# Langton's Ant!
```

Comment:
Hi @Marseonji , no need to duplicate this file under your solution folder, please remove it

Reply:
I removed it, thanks.

Reply:
Readme file in multiples folder, it was removed.